llvm/unittests/Support/KnownBitsTest.cpp

```diff
@@ -166,44 +166,82 @@ TEST(KnownBitsTest, AddCarryExhaustive) {
 }
 
 static void TestAddSubExhaustive(bool IsAdd) {
-  unsigned Bits = 4;
+  unsigned Bits = 6;
```

Comment:
Remove this change? I tend to increase it for local testing, but I don't think it actually needs to be any higher than about 3 or 4 for full test coverage.

Reply:
Ah, didn't mean to leave that in.